cdbot/constants.py

```diff
@@ -99,6 +99,11 @@ class Exchange:
 QUOTES_BOT_ID = 292953664492929025
 LOGGING_CHANNEL_ID = int(environ.get("LOGGING_CHANNEL_ID", "538494690601992212"))
 WELCOME_BOT_ID = 155149108183695360
+POLL_CHANNEL_ID = int(environ.get("POLL_CHANNEL_ID", "574878887625097227"))
+POLL_WEBHOOK_ID = int(environ.get("POLL_WEBHOOK_ID"))
+POSITIVE_EMOJI = str(environ.get("POSITIVE_EMOJI", "<:this:586246261552840705>"))
+NEUTRAL_EMOJI = str(environ.get("NEUTRAL_EMOJI", "<:somewhatthis:639201446356713502>"))
+NEGATIVE_EMOJI = str(environ.get("NEGATIVE_EMOJI", "<:notthis:485489378999926794>"))
```

Comment:
This is just worse...

Reply:
set to 🟥 and 🟨... by default but can be configured to use any emoji by setting it in the env vars.